ERCS/erc-7628.md

```diff
@@ -0,0 +1,111 @@
+    /// @param spender The address of the spender.
+    /// @return The allowance granted to the spender.
+    function allowance(uint256 tokenId, address spender) external view returns (uint256);
+
+    /// @notice Approves the specified address to spend the specified amount of tokens on behalf of the caller.
+    /// @param tokenId The identifier of the token.
+    /// @param to The address of the spender.
+    /// @param amount The amount of tokens to approve.
+    function approve(uint256 tokenId, address to, uint256 amount) external;
+
+    /// @notice Transfers tokens from the balance of one token to another.
+    /// @param _fromTokenId The identifier of the sender token.
+    /// @param _toTokenId The identifier of the recipient token.
+    /// @param amount The amount of tokens to transfer.
+    function transferFrom(uint256 _fromTokenId, uint256 _toTokenId, uint256 amount) external;
+
+    /// @notice Transfers tokens from the balance of one token to another address.
+    /// @param _fromTokenId The identifier of the sender token.
+    /// @param _to to The address of the recipient.
+    /// @param amount The amount of tokens to transfer.
+    function transferFrom(uint256 _fromTokenId, address _to, uint256 amount) external;
+
+    /// @notice Emitted when tokens are transferred from the balance of one token to another.
+    /// @param _fromTokenId The identifier of the sender token.
+    /// @param _toTokenId The identifier of the recipient token.
+    /// @param value The amount of tokens transferred.
+    event Transfer(uint256 indexed _fromTokenId, uint256 indexed _toTokenId, uint256 value);
+
+    /// @notice Emitted when an approval is granted for a spender to spend tokens on behalf of an owner.
+    /// @param _tokenId The token identifier.
+    /// @param spender The address of the spender.
+    /// @param value The amount of tokens approved.
+    event Approval(uint256 indexed _tokenId, address indexed spender, uint256 value);
+}
+```
+
+## Rationale
+
+Integrating a balance feature into each token and adopting mechanisms similar to those found in [ERC-20](./eip-20.md), we enable the seamless exchange of balances between tokens. This development allows token holders to effortlessly inquire about and transact the ownership shares represented by their token balances. Furthermore, should there be a necessity to increase the overall balance, new NFTs can be minted and their balances augmented accordingly. The omission of the `transfer` function from [ERC-20](./eip-20.md) is necessitated by the need to use `tokenId` for executing transfers. Including it would be redundant, as it overlaps with the functionality of the `transferFrom` function. Additionally, in situations where a token owner desires to transfer a balance to a specific address, a viable method involves first minting a new token for that address, then proceeding with the balance transfer, ensuring a smooth transition of value.
```

Comment:
This is definitely mixing Rationale and Motivation. The first few sentences are explaining why we need the proposal as a whole (so should be in Motivation.) The bit about omitting `transfer` is Rationale, so that can stay here.